src/components/App.vue

```diff
@@ -387,6 +396,56 @@ export default defineComponent({
       }
     });
 
+    // --- ResliceCursorWidget --- //
+    // Construct the common instance of vtkResliceCursorWidget and provide it
+    // to all the child ObliqueView components.
+    const resliceCursor = ref<vtkResliceCursorWidget>(
```

Comment:
Is it necessary for the reslice cursor widget to be a ref? If not, we can drop ref here.

Reply:
Yes, it doesn't have to be ref. Fixed it in a new commit.